tracing-subscriber/examples/fmt_stderr.rs

```diff
@@ -0,0 +1,13 @@
+#![deny(rust_2018_idioms)]
+use std::io;
+use tracing::error;
+
+fn main() {
+    let subscriber = tracing_subscriber::fmt::Subscriber::builder()
```

Comment:
Not a nit, but a clarification—do we expect that `tracing_subscriber` will eventually be re-exported from `tracing`?

Reply:
Probably not. The intention is for `tracing` to provide only the _instrumentation_ API (everything you need to instrument your code, but no subscriber implementations) while `tracing-subscriber` provides subscriber implementations & APIs for working with subscribers. 

The idea being that libraries that want to expose tracing diagnostics would depend on `tracing` only, and not have to pull in the subscriber implementations and their dependencies, while applications that set up a subscriber would depend on `tracing` (for instrumentation) and `tracing-subscriber`. Finally, subscriber _implementations_ can depend solely on `tracing-subscriber`. 

It should only be necessary to depend on `tracing-core` in very rare cases. Most of the time, nobody would depend on it directly, but factoring out the facade later would prevent multiple `tracing`/`tracing-subscriber` versions in-tree from breaking everything.